packages/clerk-js/src/ui/common/authPropHelpers.ts

```diff
@@ -78,3 +78,28 @@ export const buildAuthQueryString = (data: BuildAuthQueryStringArgs): string | n
   }
   return Object.keys(query).length === 0 ? null : qs.stringify(query);
 };
+
+export const verifyRoutingOptions = ({
```

Comment:
I think the general idea is fine! I've got two comments, so let's discuss.

1. The name is slightly misleading as we're _normalising_ the params and not just verifying them. Should we rename it accordingly?

2. You clearly have more context, but I think we only need the following check (pseudo code follows):

```tsx
props => {
  if (props.path && props.routing === undefined) {
     return { ...props, routing: 'path'} 
  }
 return props
}
```
Would that be enough?

Reply:
Typescript wise, not really. We'll get some errors there. I'd prefer if we respect the types here to avoid any future mistakes.